tool/pylib/generator/output/Script.py

```diff
@@ -128,7 +128,10 @@ def createResourceStruct(resources, formatAsTree=False, updateOnlyExistingSprite
                 for glyphname, code in res.mapping.iteritems():
                   fdsc = "@%s/%s" % (res.alias, glyphname)
                   if not fdsc in result:
-                      result[fdsc] = [result[resid][1], round(result[resid][2] / code[1]), code[0]]
+                      try:
+                          result[fdsc] = [result[resid][1], round(result[resid][2] / code[1]), code[0]]
+                      except:
+                          pass
```

Comment:
If we're getting at this point, one should start to add meaningful outputs for every potential "non inclusion" and telling why. Additionally it should output a warning for each non named glyph. Maybe this is what you wanted to say ;-)

Just before getting off topic:

From my current experience with the generator, it mostly fails with two categories: non-informative-error-messages that don't help you to find configuration problems (esp. when working with parts) or non-informative-stacktraces that need deeper inspection.

If we think we need the generator as it is for the next couple of years, I'd address some of these problems - and make it love python 2/3.